docs/reference/juju-cli/list-of-juju-cli-commands/collect-metrics.md

```diff
@@ -3,6 +3,9 @@
 > See also: [metrics](#metrics)
 
 ## Summary
+
+This functionality has been removed as of version 3.6.10.
```

Comment:
Won't this whole md file be removed whe nthe doc is generated, since the command is missing?